HaxeManual/03-type-system.tex

```diff
@@ -93,13 +93,13 @@ \subsection{Constraints}
 
 \since{4.0.0}
 
-One of the breaking changes between versions 3 and 4 is the multiple type constraint syntax. As the first example above shows, in Haxe 4 the constraints are separated by a \expr{\&} symbol instead of a comma. This is similar to the new \tref{structure extension}{types-structure-extensions} syntax.
+One of the breaking changes between versions 3 and 4 is the multiple types of constraint syntax. Like the first example above shows, in Haxe 4 the constraints are separated by a \expr{\&} symbol instead of a comma. This is similar to the new \tref{structure extension}{types-structure-extensions} syntax.
```

Comment:
No, this changes the meaning of the sentence. It is the syntax to specify multiple (simultaneous) type constraints. It is the multiple type constraint syntax.